src/app/core/models/payment-method/payment-method.mapper.ts

```diff
@@ -103,7 +103,7 @@ export class PaymentMethodMapper {
    * valid: payment methods without capabilities or which have no capabilities given in the list below
    */
   private static isPaymentMethodValid(paymentData: PaymentMethodBaseData): boolean {
-    const invalidCapabilities = ['LimitedTender', 'FastCheckout'];
+    const invalidCapabilities = ['LimitedTender'];
```

Comment:
getEligiblePaymentMethods now returns also fast checkout methods, pls write a migration note, that the selector has changed